app/src/routes/definitions.json

```diff
@@ -60,10 +60,10 @@
     "terms": "https://s3.amazonaws.com/streamr-public/streamr-terms-of-use.pdf",
     "transactions": "/core/transactions",
     "userpages": {
-        "root": "/core"
+        "root": "/core",
+        "streamPreview": "/core/stream/preview/:streamId"
```

Comment:
I would prefer `/core/stream/:id/preview`. Our urls are not that consistent, though, but that would be more in line with `/products/:id/streamPreview/:streamId` and  `/core/products/:id/edit`.

Reply:
Yep, `/core/streams/:id/preview` is the way to go. Here I just moved it over from its old location, i.e. 
```json
"userPageStreamPreview": "/core/stream/preview/:streamId",
```
I'll update!

Reply:
I'll leave marketplace's links be for now. We can address that topic later.